packages/cursorless-vscode-e2e/src/suite/fixtures/scopes/haskell/argument.actual2.scope

```diff
@@ -0,0 +1,85 @@
+abs :: Int -> Int
+abs x
+    | x >= 0    =  x
+    | otherwise = -x
+
+---
+
+[#1 Content] =
+[#1 Domain] = 0:4-0:4
+0| abs :: Int -> Int
+      ><
+1| abs x
+
+2|     | x >= 0    =  x
+
+3|     | otherwise = -x
+
+4|
+
+
+[#1 Removal] = 0:3-0:4
+0| abs :: Int -> Int
+     >-<
+1| abs x
+
+2|     | x >= 0    =  x
+
+3|     | otherwise = -x
+
+4|
+
+
+[#1 Leading delimiter] = 0:3-0:4
+0| abs :: Int -> Int
+     >-<
+1| abs x
+
+2|     | x >= 0    =  x
+
+3|     | otherwise = -x
+
+4|
+
+
+[#1 Insertion delimiter] = " "
+
+
+[#2 Content] =
+[#2 Domain] = 0:4-1:5
+0| abs :: Int -> Int
+      >-------------
+1| abs x
+   -----<
```

Comment:
this one is surprising

Reply:
Yeah, that's an odd one. Might be the same issue.